src/screens/FeedScreen/Header.js

```diff
@@ -451,7 +594,22 @@ const styles = StyleSheet.create({
   avatarImage: {height: 48, width: 48, borderRadius: 24},
   postDetail: (isPostDetail) => ({
     paddingLeft: isPostDetail ? 10 : 0
-  })
+  }),
+  contentFeedName: {
+    flexDirection: 'row'
+  },
+  textFollow: {
+    color: colors.bluePrimary,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  },
+  textFollowing: {
+    color: colors.greySubtile1,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of styles for follow/unfollow functionality is good. However, it's recommended to use a more descriptive name for these styles. For example, instead of `textFollow` and `textFollowing`, consider using `followTextStyle` and `followingTextStyle`. This makes it clear that these are styles, not text or component instances.


<!-- This is an auto-generated comment by OSS CodeRabbit -->